org.eclipse.xtext.util/src/org/eclipse/xtext/util/MergeableManifest.java

```diff
@@ -418,30 +442,1025 @@ public static String mergeIntoCommaSeparatedList(String currentString, Set<Strin
+			 * for sealing.
+			 * 
+			 * @see <a href="{@docRoot}/../specs/jar/jar.html#package-sealing">
+			 * Package Sealing</a>
+			 */
+			public static final Name SEALED = new Name("Sealed");
+
+			/**
+			 * {@code Name} object for {@code Extension-List} manifest attribute
+			 * used for the extension mechanism that is no longer supported.
+			 */
+			public static final Name EXTENSION_LIST = new Name("Extension-List");
+
+			/**
+			 * {@code Name} object for {@code Extension-Name} manifest
+			 * attribute. used for the extension mechanism that is no longer
+			 * supported.
+			 */
+			public static final Name EXTENSION_NAME = new Name("Extension-Name");
+
+			/**
+			 * {@code Name} object for {@code Extension-Installation} manifest
+			 * attribute.
+			 *
+			 * @deprecated Extension mechanism is no longer supported.
+			 */
+			@Deprecated
+			public static final Name EXTENSION_INSTALLATION = new Name("Extension-Installation");
+
+			/**
+			 * {@code Name} object for {@code Implementation-Title} manifest
+			 * attribute used for package versioning.
+			 */
+			public static final Name IMPLEMENTATION_TITLE = new Name("Implementation-Title");
+
+			/**
+			 * {@code Name} object for {@code Implementation-Version} manifest
+			 * attribute used for package versioning.
+			 */
+			public static final Name IMPLEMENTATION_VERSION = new Name("Implementation-Version");
+
+			/**
+			 * {@code Name} object for {@code Implementation-Vendor} manifest
+			 * attribute used for package versioning.
+			 */
+			public static final Name IMPLEMENTATION_VENDOR = new Name("Implementation-Vendor");
+
+			/**
+			 * {@code Name} object for {@code Implementation-Vendor-Id} manifest
+			 * attribute.
+			 *
+			 * @deprecated Extension mechanism is no longer supported.
+			 */
+			@Deprecated
+			public static final Name IMPLEMENTATION_VENDOR_ID = new Name("Implementation-Vendor-Id");
+
+			/**
+			 * {@code Name} object for {@code Implementation-URL} manifest
+			 * attribute.
+			 *
+			 * @deprecated Extension mechanism is no longer supported.
+			 */
+			@Deprecated
+			public static final Name IMPLEMENTATION_URL = new Name("Implementation-URL");
+
+			/**
+			 * {@code Name} object for {@code Specification-Title} manifest
+			 * attribute used for package versioning.
+			 */
+			public static final Name SPECIFICATION_TITLE = new Name("Specification-Title");
+
+			/**
+			 * {@code Name} object for {@code Specification-Version} manifest
+			 * attribute used for package versioning.
+			 */
+			public static final Name SPECIFICATION_VERSION = new Name("Specification-Version");
+
+			/**
+			 * {@code Name} object for {@code Specification-Vendor} manifest
+			 * attribute used for package versioning.
+			 */
+			public static final Name SPECIFICATION_VENDOR = new Name("Specification-Vendor");
+
+			/**
+			 * {@code Name} object for {@code Multi-Release} manifest attribute
+			 * that indicates this is a multi-release JAR file.
+			 *
+			 * @since 9
```

Comment:
These since tags will confuse the API tooling

Reply:
Will delete.